data/default_config/layout.json

```diff
@@ -0,0 +1,83 @@
+{
+    "dtype": "compas_viewer.configurations/LayoutConfig",
+    "data": {
+        "window": {
+            "dtype": "compas_viewer.configurations/WindowConfig",
+            "data": {
+                "about": "compas_viewer is the standalone viewer for COMPAS.",
+                "title": "COMPAS VIEWER",
+                "width": 1600,
+                "height": 900,
+                "fullscreen": false
+            },
+            "guid": "52a481b3-0a86-45df-92b1-dbd59e9cb7a6"
```

Comment:
A small tip, if `json_dump(minimal=True)` it will skip saving guids. For these configs, we don't need to have them